### ISSUE TITLE
Showing error of "Missing parameter comment"

### ISSUE THREAD

**Issue by ankitjain28may** [squizlabs/PHP_CodeSniffer#1886]
My ruleset.xml file is --
```
<?xml version="1.0"?>
<ruleset name="MyStandard">
    <description>My custom coding standard.</description>
    <rule ref="PEAR">
        <exclude name="PEAR.Commenting.FileComment.Missing"/>
        <exclude name="PEAR.Commenting.ClassComment.Missing"/>
        <exclude name="PEAR.Commenting.FunctionComment.Missing"/>
        <exclude name="PEAR.Commenting.InlineComment.Missing"/>
    </rule>
</ruleset>
```
As i have included the rule `PEAR.Commenting.FunctionComment.Missing` so it shouldn't show such errors.

**Comment by jrfnl** [comment]
You also need to exclude `Squiz.Commenting.FunctionComment.MissingParamTag`. Each error has an individual errorcode, these don't work as wildcards.

**Comment by ankitjain28may** [comment]
On including it, It still showing the same error.

**Comment by jrfnl** [comment]
@ankitjain28may In that case, I suggest you run PHPCS with the `-s` option and check to see the exact errorcode for the message you want to exclude, exclude that and try again ;-)

**Comment by ankitjain28may** [comment]
@jrfnl  Thanks, It fixed..
We explicitly write each type of missing error of FunctionComment Sniff.
```
        <exclude name="PEAR.Commenting.FunctionComment.MissingParamComment"/>
        <exclude name="PEAR.Commenting.FunctionComment.Missing"/>
        <exclude name="PEAR.Commenting.FunctionComment.MissingReturn"/>
        <exclude name="PEAR.Commenting.FunctionComment.MissingParamTag"/>
```
